workflows/mag.nf

```diff
@@ -7,7 +7,9 @@
 def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params)
 
 // Check already if long reads are provided
-include { hasExtension } from '../modules/local/functions'
+def hasExtension(it, extension) {
+    it.toString().toLowerCase().endsWith(extension.toLowerCase())
+}
```

Comment:
Could this go in a custom functions file in `lib` and be imported? Although maybe overkill for a single tiny function

Reply:
I kept it where it is now, I agree that this is not the place it is supposed to be but readability is sort of better this way I think.